Chinese/dream/dream.md

```diff
@@ -0,0 +1,7 @@
+你來到這就是在作夢~ 來吧! 以下自由選擇:
+
+1. [我要穿越](Chinese/Yifan/modernCity.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.